libs/common/common.go

```diff
@@ -62,3 +63,22 @@ func NewAWSCDKStackProps() *awscdk.StackProps {
 		},
 	}
 }
+
+// GetAllowAllVPCSecurityGroup returns a security group that allows traffic to and from the vpc over the given port
+func GetAllowAllVPCSecurityGroup(scope constructs.Construct, name *string, description *string, vpc awsec2.IVpc, port int) awsec2.SecurityGroup {
+	sg := awsec2.NewSecurityGroup(scope, name, &awsec2.SecurityGroupProps{
+		Vpc:              vpc,
+		AllowAllOutbound: jsii.Bool(true),
```

Comment:
Do you know if this can be changed from the returned object?

Reply:
Yes, it can. It's an interface with methods that invoke the API.